dm/pkg/utils/db.go

```diff
@@ -632,3 +632,20 @@ func GetTableCreateSQL(ctx context.Context, conn *sql.Conn, tableID string) (sql
 	}
 	return createStr, nil
 }
+
+// GetTableCollation gets table collation info by 'SELECE TABLE_COLLATION FROM INFORMATION_SCHEMA.TABLES'.
+func GetTableCollation(ctx context.Context, db *sql.DB, schema string, table string) (info string, err error) {
```

Comment:
use wrapped DB to retry on network problem

Reply:
ok, i will try.